### ISSUE TITLE
Don't show regulates children as suggestions in MF branch

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#2654]
Don't show 'regulates' children as suggestions in MF branch

**Comment by kimrutherford** [comment]
Is that an extension configuration change?  

There are three config lines for MF that have a regulates relation.  Should I remove them?

```
domain ID       subset relation     extension relation      range ID        Canto display text             Help text       cardinality     role
GO:0003700      is_a                has_regulation_target   TranscriptID    regulates transcription of              0,1     user
GO:0003712      is_a                has_regulation_target   ProteinID       regulates activity of transcription factor              0,1     user
GO:0030234      is_a                has_regulation_target   ProteinID       regulates activity of                   0,1     user
```

**Comment by ValWood** [comment]
Sorry I didn't explain this properly.

In  the MF branch term selection, because we show regulation children we would see (for example)

protein kinase activity
--% protein kinase regulator activity

but there is no 'inheritance' between MF terms and their regulators, and you would not really expect to see  % protein kinase regulator activity
under protein kinase activity.

This is a small thing and not urgent ( people just ignore the bonkers children)

**Comment by ValWood** [comment]
Yes, it would be for MF only. It actually means a slightly different thing here, Its confusing.

I'm surprised we use  has_part in Canto (and has_output or output_of)

**Comment by kimrutherford** [comment]
> I'm surprised we use has_part in Canto (and has_output or output_of)

Here's the discussion:
  - https://github.com/pombase/canto/issues/590

**Comment by ValWood** [comment]
OK, yes we do use. I have never been sure how sensible this is, but I have curation tracker ticket somewhere to review it.

Yes, unfortunately, we want to  show "regulates"  descendants for BP but not for MF.

It is wrong to see 
kinase regulator activity" as a  subclass of "protein kinase activity", but no hurry because it has never caused a problem.
